documentation/docs/strategies/avellaneda-market-making.md

```diff
@@ -58,43 +60,99 @@ This strategy implements the market making strategy described in the classic pap
+The closer the `risk_factor` is to **zero**, the more symmetrical will be orders will be created, and the **Reserved price** will be pretty much equal to the market mid price.
 
-The number of ticks used as a sample size for volatility calculation.
+In that case, the user is taking more inventory risk, because there will be no skew on the orders positions aiming to reach the `inventory_target_base_pct`.
 
-### `order_levels`
+The igher the value, the more aggressive the strategy will be to reach the `inventory_target_base_pct`, increasing the distance between the **Reserved price** and the market mid price.
 
-Quantity of orders to be placed on each side of the order book. 
+It's a unit-less parameter, that can be set to any non-zero value as necessary, depending on the inventory risk the user is willing to take. 
+
+> NOTE: The amount of decimal points used on the market price have a considerable influence on the how big the `risk_factor` must be to start having an effect of the **Reserved price** and on the **Optimal Spread**
+> As an example, on a market with 2 decimals (0.01), a `risk_factor = 1` can already have a noticeable effect, while on a market with 4 decimals (0.0001), the `risk_factor` must be at least around 1000 to have any noticeable effect.
+> The only way to find a value for the `risk_factor` is to experiment with different values and see it's effects on the **Reserved price** and the **Optimal spread**.
+
+Given the right market conditions and the right `risk_factor`, it's possible that the optimal spread will be wider than the absolute price of the asset, or that the reserved price will by far away from the mid price, in both cases resulting in the optimal bid price to be lower than or equal to 0. If this happens neiher buy or sell will be placed. To prevent it from happening, users can set the `risk_factor` to a lower value.
+
+#### ETA (Order size adjustment)
+
+If users choose to set the `eta` parameter, order sizes will be adjusted to further optimize the strategy behavior in regards to the current and desired portfolio allocation.
+
+With a value of `eta = 1`, buy and sell orders will have the same size. A different value will create assymetrical order sizes, with the goal to reach the `inventory_target_pct` faster.
+
+#### Order levels
+
+Users have an option to layer orders on both sides. If more than 1 `order_levels` are chosen, multiple buy and sell limit orders will be created on both sides, with predefined price distances from each other, with the levels closest to the reserved price being set to the optimal bid and ask prices. This price distance between levels is defined as a percentage of the optimal spread calculated by the strategy. The percentage is given as the `level_distances` parameter. Given that optimal spreads tend to be tight, the `level_distances` values should be in general in tens or hundreds of percents.
+
+### Trading logic flow
+
+![Figure 1: Strategy flow chart](/assets/img/avellaneda.svg)
```

Comment:
Add a section that explains the diagram at a high level by adding a sentence for each step. For example, for "Are characteristics estimated?" add a sentence which characteristics are estimated and by what process.